trivy/trivy-secret.yaml

```diff
@@ -0,0 +1,61 @@
+# This file is part of kuberpult.
+
+# Kuberpult is free software: you can redistribute it and/or modify
+# it under the terms of the Expat(MIT) License as published by
+# the Free Software Foundation.
+
+# Kuberpult is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# MIT License for more details.
+
+# You should have received a copy of the MIT License
+# along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>.
+
+# Copyright freiheit.com
+
+# These allow rules are enabled by default. We shall disable all allow rules unless explicitly enabled.
+# The list below consists of all the built-in allow rules. It can be found at [1, 2].
+# 
+# [1] - https://aquasecurity.github.io/trivy/v0.51/docs/scanner/secret/
+# [2] - https://github.com/aquasecurity/trivy/blob/main/pkg/fanal/secret/builtin-allow-rules.go
+disable-allow-rules:
+- "tests"
```

Comment:
Why disable tests exclusions if you are manually adding exclusions for specific cases in test files below?